feature-engineering/deep_feature_extractor.md

```diff
@@ -6,11 +6,9 @@ as input to train another classifier such as a LogisticClassifier,
 SVMClassifier, BoostedTreesClassifier, or NeuralNetClassifier
 
 Deep features can be used to extract features from your own models or using a
-pre-trained model for ImageNet, as described by Alex Krizhevsky et. al.
-avaliable for use at
-http://s3.amazonaws.com/dato-datasets/deeplearning/imagenet_model_iter45.  If
-`model='auto'` is used, we chose an appropriate model from our batch of
-pre-trained models.
+pre-trained model for ImageNet (NIPS 2012, Alex Krizhevsky et al.).  Dato provides 
+a free pre-trained model for use as demonstrated below. If `model='auto'` is used, 
+we chose an appropriate model from our batch of pre-trained models.
```

Comment:
The first person (we/our) perspective is an odd phrasing.  How about 

If `model='auto'` is used, an appropriate mode is chosen from a batch of pre-trained models.